tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt

```diff
@@ -1,6 +1,6 @@
-# Tag: nvidia/cuda:11.6.1-cudnn8-devel-ubuntu20.04
-# Label: com.nvidia.cuda.version: 11.6.1
-# Label: com.nvidia.cudnn.version: 8.4.0
+# Tag: nvidia/cuda:11.8.0-cudnn8-devel-ubuntu20.04
```

Comment:
Which pipeline needs this file?

Reply:
Hi it's Linux Multi GPU TensorRT CI Pipeline. Check this [run_dockerbuild.sh](https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/run_dockerbuild.sh#L98-L100) for more detail